app/actors/SessionsScheduler.scala

```diff
@@ -233,8 +233,11 @@ class SessionsScheduler @Inject()(sessionsRepository: SessionsRepository,
 
     scheduledEmails = scheduledEmails - key
 
-    emailManager ! EmailActor.SendEmail(
-      emails, fromEmail, "Feedback reminder", views.html.emails.reminder(emailInfo, feedbackUrl).toString)
+    sessions.foreach{ session =>
```

Comment:
With these changes multiple emails are going to be sent to everyone, we only want a single email to be sent.